client/src/components/Searchbar.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+export default function Searchbar({ getQuery }) {
+  const [text, setText] = useState('');
+  const onChange = (q) => {
```

Comment:
What does `q`stand for? `query`? Probably an unnecessary abbreviation, no need to save characters :)